commitizen/cz/conventional_commits/conventional_commits.py

```diff
@@ -110,16 +109,14 @@ def questions(self) -> List[Dict[str, Any]]:
                 "name": "subject",
                 "filter": parse_subject,
                 "message": (
-                    "Subject. Concise description of the changes. "
-                    "Imperative, lower case and no final dot:\n"
+                    "Write a short and imperative summary of the code changes:\n"
```

Comment:
Why removing the lower case and final dot? I think it's a way to "educate" without forcing it